sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableConfig.java

```diff
@@ -69,6 +69,9 @@ abstract class BigtableConfig implements Serializable {
   /** Bigtable emulator. Used only for testing. */
   abstract @Nullable String getEmulatorHost();
 
+  /** Reporting throttle time to Dataflow flag. */
+  abstract boolean getBulkMutationDataflowThrottling();
```

Comment:
I think the name is a bit too long, maybe change it to `getBulkMutationThrottling` and add an `@Experimental` flag and a comment saying it's experimental feature.

And also comment should be something like: whether throttling time is reported to dataflow jobs